lang/en/tool_excimer.php

```diff
@@ -48,6 +48,7 @@
 $string['request_ms_desc'] = 'Record a profile only if it runs at least this long.';
 $string['num_slowest_by_page'] = 'Max to save by page';
 $string['num_slowest_by_page_desc'] = 'Only the N slowest profiles will be kept for each script page.';
+$string['noexcimerprofiler'] = 'ExcimerProfiler class does not exist so profiling cannot continue.';
```

Comment:
Can you add some words to help someone fix the issue, even if it's just a link to:

https://github.com/catalyst/moodle-tool_excimer#installation

Reply:
Sure thing. I had it that way because though running through the install instructions I only resolved it after a container restart. Hopefully not always the case, but I'll add it to maybe a troubleshooting section as well.